docs/resources/aviatrix_edge_neo_ha.md

```diff
@@ -0,0 +1,79 @@
+---
+subcategory: "Multi-Cloud Transit"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_edge_neo_ha"
+description: |-
+  Creates an Aviatrix Edge NEO HA
+---
+
+# aviatrix_edge_neo_ha
+
+The **aviatrix_edge_neo_ha** resource creates the Aviatrix Edge NEO HA.
+
+## Example Usage
+
+```hcl
+# Create an Edge NEO HA
+resource "aviatrix_edge_neo_ha" "test" {
+  primary_gw_name = "primary_edge_neo"
+  device_id       = "device5678"
+
+  interfaces {
+    name          = "eth0"
+    type          = "WAN"
+    ip_address    = "10.230.5.32/24"
+    gateway_ip    = "10.230.5.100"
+    wan_public_ip = "64.71.24.221"
+  }
+
+  interfaces {
+    name       = "eth1"
+    type       = "LAN"
+    ip_address = "10.230.3.32/24"
+  }
+
+  interfaces {
+    name        = "eth2"
+    type        = "MANAGEMENT"
+    enable_dhcp = false
+    ip_address  = "172.16.15.162/20"
+    gateway_ip  = "172.16.0.1"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+### Required
+* `primary_gw_name` - (Required) Primary Edge NEO name.
+* `device_id` - (Required) Device ID.
+* `interfaces` - (Required) WAN/LAN/MANAGEMENT interfaces.
+    * `name` - (Required) Interface name.
+    * `type` - (Required) Type.
+    * `bandwidth` - (Optional) Bandwidth.
```

Comment:
what is bandwidth? What type of values are accepted? Description isnt clear, esp since its not shown in the example code either